packages/twenty-front/src/modules/object-metadata/components/ObjectMetadataNavItems.tsx

```diff
@@ -14,6 +16,9 @@ export const ObjectMetadataNavItems = () => {
   const currentPath = useLocation().pathname;
 
   const { records: views } = usePrefetchedData<View>(PrefetchKey.AllViews);
+  const loading = useIsPrefetchLoading();
+
+  if (loading) return <ObjectMetadataNavItemsSkeletonLoader />;
```

Comment:
let's avoid if () return and use if () { return }

Reply:
going to check if we can have eslint enfore this